bibliopixel/drivers/driver_base.py

```diff
@@ -5,7 +5,15 @@
 
 
 class DriverBase(object):
-    """Base driver class to build other drivers from."""
+    """
+    Base driver class to build other drivers from.
+
+    :param int num: Number of total pixels held by this driver
+    :param int width: Width of matrix, for use with :py:class:`bibliopixel.layout.matrix.Matrix`
```

Comment:
The horror!  :-D :-P 

(This isn't an actionable comment, btw... just a reminder of how the "width height" thing appears in more than one place when it should DRY.)